easybuild/easyconfigs/s/spglib/spglib-1.9.9-intel-2017b.eb

```diff
@@ -0,0 +1,38 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   David Quigley (University of Warwick)
+# License::   MIT/GPL
+# $Id$
+#
+##
+easyblock = 'CMakeMake'
+name = 'spglib'
+version = '1.9.9'
+
+homepage = 'https://atztogo.github.io/spglib/'
+description = """Spglib is a library for finding and handling crystal symmetries written in C."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+builddependencies = [
+    ('CMake', '3.9.5'),
+]
+
+source_urls = ['https://github.com/atztogo/spglib/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['315642d356c3e7b9cd0e3854f39e10cae89d5f757860db280f61eab3b70d17c2']
+
+separate_build_dir = True
+
+# Build places libs and headers in the source directory - copy them manually
+installopts = ' && cp -r ../%(name)s-%(version)s/lib ../%(name)s-%(version)s/include %(installdir)s/'
+
+parallel = 1
+
+sanity_check_paths = {
+    'files': ['lib/libsymspg.so.%(version)s', 'lib/libsymspg.a'],
```

Comment:
@dquigley-warwick - can you explain the change here? Normally we use `SHLIB_EXT`, as it was in the older versions.

Reply:
No (good) reason. I'll update the PR.